ansible/roles/init-dev-configuration/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+---
+- name: copy CS container configuration files
+  copy:
+    src: "files/cs/"
+    dest: "/"
+    group: "root"
+    owner: "root"
+    mode: "0644"
+    directory_mode: "0755"
+  when: inventory_hostname in groups['cs_servers']
+  tags: dev-configuration
+
+- name: copy SS container configuration files
+  copy:
+    src: "files/ss/"
+    dest: "/"
+    group: "root"
+    owner: "root"
+    mode: "0644"
+    directory_mode: "0755"
+  when: inventory_hostname in groups['ss_servers']
+  tags: dev-configuration
+
+- name: Install and configure Mailpit
+  when: inventory_hostname in groups['ss_servers']
```

Comment:
Depends on how you look at it :) One might say that each CS, SS is already isolated as in prod they would be separated. But I guess we can move mailpit to CS instance, that way it will be a single instance. Or you had different deployment ideas?

Reply:
IS and CA are options too, but I think CS is fine. Maybe it could be moved to separate role?